datarepo-clienttests/src/main/resources/servers/rcdev.json

```diff
@@ -12,7 +12,9 @@
   "containerName": "rc-jade-datarepo-api",
   "deploymentScript": {
     "name": "ModularHelmChart",
-    "parameters": ["https://raw.githubusercontent.com/broadinstitute/datarepo-helm-definitions/master/dev/rc/datarepo-api.yaml"]
+    "parameters": ["file:///Users/aurora/Desktop/repositories/datarepo-helm-definitions/dev/rc/datarepo-api.yaml"]
```

Comment:
Do we ever actually read this file?  Probably don't want this pointing to your local machine